src/openrct2-android/app/src/main/CMakeLists.txt

```diff
@@ -124,13 +140,61 @@ set_target_properties(ssl PROPERTIES IMPORTED_LOCATION
 )
 add_dependencies(ssl libs)
 
+add_library(crypto SHARED IMPORTED)
+set_target_properties(crypto PROPERTIES IMPORTED_LOCATION
+    ${CMAKE_BINARY_DIR}/libs/lib/${CMAKE_SHARED_LIBRARY_PREFIX}crypto${CMAKE_SHARED_LIBRARY_SUFFIX}
+)
+add_dependencies(crypto libs)
+
+add_library(vorbis SHARED IMPORTED)
+set_target_properties(vorbis PROPERTIES IMPORTED_LOCATION
+    ${CMAKE_BINARY_DIR}/libs/lib/${CMAKE_SHARED_LIBRARY_PREFIX}vorbis${CMAKE_SHARED_LIBRARY_SUFFIX}
+)
+add_dependencies(vorbis libs)
+
+add_library(vorbisfile SHARED IMPORTED)
+set_target_properties(vorbisfile PROPERTIES IMPORTED_LOCATION
+    ${CMAKE_BINARY_DIR}/libs/lib/${CMAKE_SHARED_LIBRARY_PREFIX}vorbisfile${CMAKE_SHARED_LIBRARY_SUFFIX}
+)
+add_dependencies(vorbisfile libs)
+
+add_library(FLAC SHARED IMPORTED)
+set_target_properties(FLAC PROPERTIES IMPORTED_LOCATION
+    ${CMAKE_BINARY_DIR}/libs/lib/${CMAKE_SHARED_LIBRARY_PREFIX}FLAC${CMAKE_SHARED_LIBRARY_SUFFIX}
+)
+add_dependencies(FLAC libs)
+
+add_library(brotlidec SHARED IMPORTED)
+set_target_properties(brotlidec PROPERTIES IMPORTED_LOCATION
+    ${CMAKE_BINARY_DIR}/libs/lib/${CMAKE_SHARED_LIBRARY_PREFIX}brotlidec${CMAKE_SHARED_LIBRARY_SUFFIX}
+)
+add_dependencies(brotlidec libs)
+
+add_library(brotlicommon SHARED IMPORTED)
+set_target_properties(brotlicommon PROPERTIES IMPORTED_LOCATION
+    ${CMAKE_BINARY_DIR}/libs/lib/${CMAKE_SHARED_LIBRARY_PREFIX}brotlicommon${CMAKE_SHARED_LIBRARY_SUFFIX}
+)
+add_dependencies(brotlicommon libs)
+
+add_library(ogg SHARED IMPORTED)
+set_target_properties(ogg PROPERTIES IMPORTED_LOCATION
+    ${CMAKE_BINARY_DIR}/libs/lib/${CMAKE_SHARED_LIBRARY_PREFIX}ogg${CMAKE_SHARED_LIBRARY_SUFFIX}
+)
+add_dependencies(ogg libs)
+
+add_library(bz2 SHARED IMPORTED)
+set_target_properties(bz2 PROPERTIES IMPORTED_LOCATION
+    ${CMAKE_BINARY_DIR}/libs/lib/${CMAKE_SHARED_LIBRARY_PREFIX}bz2${CMAKE_SHARED_LIBRARY_SUFFIX}
+)
+add_dependencies(bz2 libs)
+
 
 include_directories(SYSTEM "${CMAKE_BINARY_DIR}/libs/include")
 include_directories(SYSTEM "${CMAKE_BINARY_DIR}/libs/include/freetype2")
 include_directories(SYSTEM "${CMAKE_BINARY_DIR}/libs/include/SDL2")
 
 # now build app's shared lib
-add_definitions(-DDISABLE_DISCORD_RPC -DDISABLE_HTTP -DDISABLE_NETWORK -DDISABLE_FLAC -DDISABLE_VORBIS -DDISABLE_OPENGL -DGL_GLEXT_PROTOTYPES -D__STDC_LIMIT_MACROS -DNO_TTF -DSDL_MAIN_HANDLED)
+add_definitions(-DDISABLE_DISCORD_RPC -DDISABLE_OPENGL -DGL_GLEXT_PROTOTYPES -D__STDC_LIMIT_MACROS -DNO_TTF -DSDL_MAIN_HANDLED)
```

Comment:
If you've added freetype, do you still need `-DNO_TTF` ?

Reply:
It's still needed, I need to flesh out https://github.com/OpenRCT2/OpenRCT2/blob/3f81a491b0ac9cc45730ae42fbd3194abe51aa2d/src/openrct2/platform/Platform.Android.cpp#L154-L160

Reply:
I filed https://github.com/OpenRCT2/OpenRCT2/issues/21634 to keep track of it